test/spec/invertedIndexSpec.js

```diff
@@ -97,77 +85,114 @@ describe('InvertedIndex', () => {
             an: [false, true, true],
             unusual: [false, true, true],
             alliance: [false, true, true],
-          }
+          },
+          filename: 'shortStories.json'
         }
       );
     });
-    it('should return false if file index was not found', () => {
-      expect(index.getIndex('notIndex.json')).toBeFalsy();
+
+    it('correctly stores all words in the file', () => {
+      expect(index[filename].allWords).toEqual(
+        ['alice', 'falls', 'into', 'an', 'unusual', 'alliance']
+      );
     });
   });
 
-  describe('SearchIndex', () => {
-    it('should return false if filename does not exist', () => {
-      expect(index.searchIndex('alice', 'wrongData.json')).toBeFalsy();
-    });
-    it('should return return the correct search result', () => {
-      expect(typeof index.searchIndex('alice', this.filename)).toEqual('object');
-      expect(index.searchIndex('alice hahaha unusual', this.filename)).toEqual(
+  describe('GetIndex', () => {
+    it('should return the correct index', () => {
+      expect(typeof index.getIndex('shortStories.json')).toEqual('object');
```

Comment:
I think this can be moved to another test case checking that getIndex actually returns an object